### PR TITLE
Add support for Restore controller

### PR DESCRIPTION
**Pull request by AmitRoushan**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind new feature
> /kind bug fix
> /kind cleanup
> /kind revert change
> /kind design
> /kind documentation
> /kind enhancement

/kind new feature

**What this PR does / why we need it**:
The PR enable kahu with metadata restore functionality

**Which issue(s) this PR fixes**:

Fixes #

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED
/kind TESTED
**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

File: providerframework/metaservice/lib/go/client.go [hunk 1]
```diff
@@ -15,10 +15,25 @@
 package metaservice
 
 import (
+	"net"
+
 	"google.golang.org/grpc"
 )
 
-func NewLBDial(target string, opts ...grpc.DialOption) (*grpc.ClientConn, error) {
+func newLBDial(target string, opts ...grpc.DialOption) (*grpc.ClientConn, error) {
```
Comment (sushanthakumar) [inline providerframework/metaservice/lib/go/client.go:23]: This function should be exported as it is used outside the package

Comment (AmitRoushan) [reply providerframework/metaservice/lib/go/client.go]: updated code as per comment

**Review by sushanthakumar — APPROVED** [review]
LGTM

**Review by PravinRanjan10 — APPROVED** [review]
LGTM

**Pull request merged**